File: despatch.Domain/Models/Default/Response/UserRoomsResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class UserRoomsResponse : BaseEntityResponse
    {
        public string MissedMessagesCount { get; set; }

        public string UserId { get; set; }
        public string RoomId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual RoomResponse Room { get; set; }

    }
}

File: despatch.WEB/app/src/swagger/models/user-rooms-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { RoomResponse } from './room-response';
import { UserResponse } from './user-response';
export interface UserRoomsResponse extends BaseEntityResponse{
  missedMessagesCount?: string;
  room?: RoomResponse;
  roomId?: string;
  user?: UserResponse;
  userId?: string;
}
